include/exiv2/jpgimage.hpp

```diff
@@ -91,28 +91,18 @@ namespace Exiv2 {
                   3 if no data for psTag was found in pPsData;<BR>
                  -2 if the pPsData buffer does not contain valid data.
         */
-        static int locateIrb(const byte *pPsData,
```

Comment:
This is the function signature that is later implemented, thus I do not like this change.    
IMHO it's more confusing if the two differ.

Reply:
Fixed.

Reply:
`sizeHdr` and `sizeData` are pointers to an output variable. Thus having them `uint32* const` makes sense from the standpoint, that one wouldn't want to accidentally change that pointer and then write the output to a different place.  


So the suggested change here, IMHO, makes the interface less secure.   
If you would like to clean this up, I think moving this to use references would be great :+1: